docs/content/en/docs/tasks/cluster/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -16,6 +16,21 @@ The upgrade command also upgrades core components of EKS Anywhere and lets the u
 >**_NOTE:_** Currently only Minor Version Upgrades are support for Bare Metal clusters. No other aspects of the cluster upgrades are currently supported.
 >
 
+{{% alert title="Important" color="warning" %}}
+
+In `eksctl anywhere` version `v0.15.0`, we introduced the full lifecycle controller to fully manage new workload clusters.
```

Comment:
We have v0.15.0 here but v0.13.0 in the vsphere/cloudstack upgrade page. Which one is it?